src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor/ClassifiedSpan.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor;
+
+internal readonly record struct ClassifiedSpan(
```

Comment:
Curious: did you write a tool to generate these types or just do it all by hand?

Reply:
All by hand. There aren't enough types and members to bother writing a tool. Plus, I figured that it's a one-time event.